doc/release_notes.rst

```diff
@@ -32,39 +23,1096 @@ This version 0.1 release has the following software components:
 * The ACRN Block, NIC, and console Virtio drivers
 * The ACRN Virtio Backend Service(VBS) and the Virtio and Hypervisor Service Module (VHM).
 
-Version 0.1 features include:
-
-- Acrnctl
-- Acrntrace
-- Acrnlog
-- SOS lifecycle
-- vSBL
-- virtio-blk
-- virtio-net
-- USB pass-thru
-- CSE pass-thru
-- IOC sharing (incl. cbc attach, cbc driver)
-- IPU pass-thru
-- BT pass-thru
-- SD card pass-thru
-- audio pass-thru
-- surface sharing
-- multi-plane, multi-pipe
-- HDMI
-- eDP
+Version 0.1 new features
+========================
+
+* In addition to the Apollo Lake NUC, the Apollo Lake UP Squared (UP2)
+  board is supported. (See :ref:`hardware` for supported platform
+  details.)
+
+* Virtual Graphics support added:
+
+  - GVT-g for ACRN (a.k.a ACRNGT): GPU sharing feature enabled. Service OS
+    and User OS can run GPU workload simultaneously.
+  - Direct display supported by ACRNGT. Service OS and User OS are each
+    assigned to different display. The display ports supports eDP, HDMI.
+  - Surface sharing supported by ACRNGT. SOS can access an individual
+    surface or a set of surfaces from UOS without having to access the
+    entire frame buffer of UOS.
+  - See :ref:`APL_GVT-G-hld` documentation for more information.
+
+* Virtio network supported. SOS and UOS can share physical LAN network
```

Comment:
How about rephrasing this to "Virtio standard supported". At the moment, the first sentence talks about the network but the rest of the paragraph talks about both network and device storage (which really is what virtio is about, i.e. both network and device storage).